### DISCUSSION TITLE
some using questions

### DISCUSSION THREAD

**Discussion by webyang-male** [facebook/docusaurus#7523] [General] [answered]
- Why does the documentation blog at least only display the No. 3 title directory in the sidebar? How can I display the h1~h6 directory normally?

为什么文档博客最低只在侧边栏显示3号标题目录？如何才能正常显示h1~h6目录？
- How to import custom js files?

怎么引入自定义的js文件？

- How to open submenus of custom navigation menu?

怎么开启自定义导航菜单的子菜单？

Newcomers don't understand these configurations when they read the documentation, and I very much hope to get help.

**Comment by Josh-Cena** [comment, marked as answer]
> Why does the documentation blog at least only display the No. 3 title directory in the sidebar? How can I display the h1~h6 directory normally?

https://docusaurus.io/docs/next/api/themes/configuration#table-of-contents

It only shows h2/h3 headings, because that's supposed to be the most "structural". MDN docs even only shows h2 headings.

> How to import custom js files?

Are these separate scripts or are supposed to be integrated with the main bundle? You can either use [`script` tags](https://docusaurus.io/docs/api/docusaurus-config#scripts) or [client modules](https://docusaurus.io/docs/advanced/client#client-modules), depending on the use-case.

> How to open submenus of custom navigation menu?

I don't understand that. Do you have an example?

**Reply by webyang-male** [reply]
Thank you very much for your patient reply. Regarding the third question, I have put the screenshot below, I just want to implement a secondary navigation bar function
![image](https://user-images.githubusercontent.com/52821367/170976245-d2581546-054e-4a97-9078-6632cdbdd5e9.png)

**Reply by Josh-Cena** [reply]
There's nothing as nested dropdowns yet. See https://github.com/facebook/docusaurus/issues/5938

**Reply by webyang-male** [reply]
Thanks!Hope the project gets better and better！
